README.md

```diff
@@ -476,14 +479,18 @@ Anatomy of a Redis command
 
 All the Redis commands are defined in the following way:
 
-    void foobarCommand(client *c) {
-        printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
-        addReply(c,shared.ok); /* Reply something to the client. */
-    }
+```c
+void foobarCommand(client *c) {
+    printf("%s",c->argv[1]->ptr); /* Do something with the argument. */
+    addReply(c,shared.ok); /* Reply something to the client. */
+}
+```
 
 The command is then referenced inside `server.c` in the command table:
```

Comment:
i am not sure about the others, but i can see this one now is outdated (in 7.0, it is inside commands.c or commands/xxx.json)

Reply:
Thanks, I will track down the reference and update the PR accordingly.